uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1031.php

```diff
@@ -22,7 +22,9 @@ private function up1031_updateManagedEntitiesTable() {
       'return' => ['id'],
       'name' => 'Application',
     ]);
-
+    if($caseType['count']===0){
```

Comment:
You're missing some spaces here

```
if ($caseType['count'] === 0){
```

I think it would be clearer to have an empty line before and after this if statement too